params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
So it's better unless there's another file that needs to be 'first' for some reason. In which case we are back to the same system  we have right now. I find it highly unlikely that we will have other settings that can't go into base hardware that needs to be a first include.